src/api.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/prefer-default-export */
+
+export const get = async (url, endPoint) => {
+  const result = await fetch(`${url}${endPoint}`);
+  return result.json();
+};
+
+export const post = async (url, endPoint, body) => {
+  const result = await fetch(`${url}${endPoint}`, {
+    method: 'POST',
+    body,
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  return result;
```

Comment:
- Solve errors in your with your API:
![Screen Shot 2021-08-05 at 07 33 58](https://user-images.githubusercontent.com/22258274/128350687-6160d8ee-8712-4ee1-9fa6-d4908d472ccf.png)

Reply:
On our end, access to fetch at Rick and Morty Api is not blocked by the core policy,
![Screenshot from 2021-08-05 15-39-00](https://user-images.githubusercontent.com/57562869/128360607-bd6dc5b9-5b4b-4eb4-bd9e-437fa929023b.png)

However, we added the Access-control-origin on the post request.